index.bs

```diff
@@ -1153,7 +1154,9 @@ The dictionary SHALL have the following attributes:
 :: The number of bytes sent on the QUIC connection, including retransmissions.
    Does not include UDP or any other outer framing.
 : <dfn for="WebTransportStats" dict-member>packetsSent</dfn>
-:: The number of packets sent on the QUIC connection, including retransmissions.
+:: The number of packets sent on the QUIC connection, including loss.
+: <dfn for="WebTransportStats" dict-member>packetsLost</dfn>
+:: The number of packets lost on the QUIC connection.
```

Comment:
As above.

Plus, I would note that this value, unlike `packetsSent` does not monotonically increase as packets that are determined to be lost can be "found" when they are later successfully acknowledged.